scadalts-ui/src/store/scripts/index.js

```diff
@@ -1,20 +1,94 @@
-const storeEvents = {
+const storeScripts = {
 	state: {},
 
 	mutations: {},
 
 	actions: {
-		fetchScripts({ dispatch }, payload) {
+		searchScripts({ dispatch }, payload) {
+			return dispatch('requestGet', `/scripts/search`);
+		},
+		runScript({ dispatch }, xid) {
+			let url = `/scripts/execute/${xid}`;
+			return dispatch('requestPost', { url });
+		},
+		async validateXid({ dispatch }, payload) {
+			let url = `/scripts/validateXid`;
+			return dispatch('requestPost', {
+				url,
+				data: {
+					id: payload.id,
+					userId: payload.userId,
+					xid: payload.xid,
+					name: payload.name,
+					script: payload.script,
+					pointsOnContext: payload.pointsOnContext,
+					datapointContext: payload.datapointContext,
+					datasourceContext: payload.datasourceContext,
+				},
+			});
+		},
+		async validateScriptBody({ dispatch }, payload) {
+			alert(JSON.stringify('PL'));
```

Comment:
I found that this alert is blocking typing the script body...

Reply:
fixed